pkg/gui/presentation/files_test.go

```diff
@@ -37,6 +38,16 @@ func TestRenderFileTree(t *testing.T) {
 			},
 			expected: []string{" M test"},
 		},
+		{
+			name: "line changes",
+			files: []*models.File{
+				{Name: "test", ShortStatus: " M", HasStagedChanges: true, LinesAdded: 1, LinesDeleted: 1},
+				{Name: "test2", ShortStatus: " M", HasStagedChanges: true, LinesAdded: 1},
+				{Name: "test3", ShortStatus: " M", HasStagedChanges: true, LinesDeleted: 1},
+			},
+			showLineChanges: true,
+			expected:        []string{" M test +1 -1", " M test2 +1", " M test3 -1"},
```

Comment:
Formatting nitpick: I'd prefer these to be on separate lines:
```
			expected: []string{
				" M test +1 -1",
				" M test2 +1",
				" M test3 -1",
			},
```
Also, maybe add one where LinesAdded and LinesDeleted are both 0?